src/gui/transferlistwidget.h

```diff
@@ -118,11 +118,14 @@ private slots:
     void askNewCategoryForSelection();
     void saveSettings();
 
-private:
+protected:
     void dragEnterEvent(QDragEnterEvent *event) override;
     void dragMoveEvent(QDragMoveEvent *event) override;
     void dropEvent(QDropEvent *event) override;
     void wheelEvent(QWheelEvent *event) override;
+    void mousePressEvent(QMouseEvent *event) override;
```

Comment:
I would implement this workaround using an event filter instead of changing the `TransferListWidget` class itself.